docs/src/getting-started/uninstallation.md

```diff
@@ -0,0 +1,65 @@
+# Uninstallation
+## Linux & macOS
+Use the `asdf` docs to learn how to remove the Starknet Foundry plugin [here](https://asdf-vm.com/manage/plugins.html#remove)
+
+To ensure Starknet Foundry is fully removed, run:
+
+```bash
+snforge --version
+```
+
+If the command is not found, the uninstallation was successful.
+
+## Windows
+### Remove Starknet Foundry Files
+You can delete Starknet Foundry manually by navigating to:
+
+```
+📂 C:\Users\YourUsername\AppData\Local\Programs\snfoundry
+```
+
+Right-click the folder and select Delete.
+
+### Remove Starknet Foundry from System PATH
+If you added snfoundry/bin to your system PATH, you need to remove it:
+
+For Windows 10 & 11:
+
+1.	Open the Start Menu, search for “Environment Variables”, and open Edit the system environment variables.
+2.	In the System Properties window, click Environment Variables.
+3.	Under System variables (or User variables), find the Path entry and select Edit.
+4.	Look for an entry similar to:
+
```

Comment:
Again, I'd just link to the specific installation section. We link to an article that explain how setting path works there, I'd prefer we don't keep any instructions on modifying it ourselves.